src/components/editor/ResourceTemplate.jsx

```diff
@@ -51,6 +52,7 @@ ResourceTemplate.propTypes = {
 
 const mapStateToProps = state => ({
   resourceTemplate: rootResource(state),
+  title: state.selectorReducer.editor.rootResourceTitle,
```

Comment:
so it didn't work as

`state.selectorReducer.entities.resourceTemplates[rtId].label ?`

cause that would avoid the addition of something to editor in redux state

Reply:
Let me try that. I think I missed `rtId` when I tried this the first time.

Reply:
Nope, getting related syntax errors `rtId is unknown` when I try that. Happy to pair on this if preferred, but I'm clear on how to make that work in this case.

Reply:
DONE :-)